### PR TITLE
UPSTREAM: 72980: Fix Cinder volume limits

### PR DESCRIPTION
**Pull request by gnufied**
Partly fixes https://bugzilla.redhat.com/show_bug.cgi?id=1669544 .This predicates needs to be explicitly enabled by scheduler operator in openstack environments (or may be by default).

/sig storage
/sig scheduling

### CODE REVIEW

File: vendor/k8s.io/kubernetes/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go [hunk 1]
```diff
@@ -835,6 +835,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 			{"name": "MaxEBSVolumeCount"},
 			{"name": "MaxGCEPDVolumeCount"},
 			{"name": "MaxAzureDiskVolumeCount"},
+                        {"name": "MaxCinderVolumeCount"},
```
Comment (tsmetana) [inline vendor/k8s.io/kubernetes/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go]: Looks like the indentation is off a bit...

**Comment by jsafrane** [comment]
Should be the predicate enabled by default? I don't see it enabled in https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/scheduler/algorithmprovider/defaults/defaults.go#L106

**Comment by gnufied** [comment]
@jsafrane yeah the predicate is not enabled by default. This was intentional and done so in according to discussion with sig-scheduling.

**Comment by jsafrane** [comment]
/lgtm
(bad json formatting is present also in usptream PR)

File: vendor/k8s.io/kubernetes/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go (hunk 1 above)
Comment (gnufied) [reply vendor/k8s.io/kubernetes/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go]: is this github thing? I don't see indentation being off....

screenshot from 2019-02-08 11-19-55

Comment (gnufied) [reply vendor/k8s.io/kubernetes/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go]: Alright you nitpickers - I did not use tabs for indenting this one and use whitespaces and that is why indentation looked bit off. Should be fixed. :-)

**Comment by jsafrane** [comment]
/assign @mfojtik @smarterclayton 
for approval of OCP blocker

**Comment by sjenning** [comment]
weird
`error: unable to create a release: operator "service-ca-operator" failed to map images: image file "/tmp/release-image-0.0.1-2019-03-13-143827483206812/service-ca-operator/image-references" referenced image "service-ca" that is not part of the input images`

**Pull request merged**